.gitignore

```diff
@@ -1,5 +1,13 @@
 _build/
 _site/
+Properties/
+node_modules/
+wwwroot/
+
+#override .sln when required. Not needed in most cases.
+*.sln
+
+Project_Readme.html
```

Comment:
I would rather not include .sln here.  There are times when we need it, and it's simple enough to omit it if we don't need it, and it doesn't hurt anything if it's there when we don't need it.